src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -81,9 +81,12 @@ public SelectorItem()
 			// of a PointerPressed is a child that captures the pointer and may handle Pointer<Released/Exited|Canceled>
 			// without handling PointerPressed. In that case we wouldn't receive a PointerPressed without
 			// a matching Pointer<Released|Exited|Canceled> without handledEventsToo.
+			// Note: we also subscribe to PointerCaptureLostEvent to cleanup state also when pointer is canceled by scroller
+			//		 (i.e. with CanceledByDirectManipulation, in that case we only raise a CaptureLost event, not cancelled).
```

Comment:
```suggestion
			// of a PointerPressed is a child that captures the pointer and may handle Pointer<Released/Exited|Canceled>
			// without handling PointerPressed. In that case, we wouldn't receive a PointerPressed without
			// a matching Pointer<Released|Exited|Canceled> without handledEventsToo.
			// Note: we also subscribe to PointerCaptureLostEvent to cleanup the state when the pointer is canceled by the scroller
			//		 (i.e. with CanceledByDirectManipulation, in that case, we only raise a CaptureLost event, not canceled).
```

Reply:
Won't re-run build for a typo, will include this in https://github.com/unoplatform/uno/pull/13968 (which also contains this fix)